app/index.html

```diff
@@ -177,14 +177,16 @@ <h4 class="mdl-cell mdl-cell--12-col">Details</h4>
             <div class="mdl-cell mdl-cell--12-col">
               <h4>Features</h4>
               Minim duis incididunt est cillum est ex occaecat consectetur. Qui sint ut et qui nisi cupidatat. Reprehenderit nostrud proident officia exercitation anim et pariatur ex.
-              <ul class="toc">
-                <h4>Contents</h4>
-                <a href="#lorem1">Lorem ipsum</a>
-                <a href="#lorem2">Lorem ipsum</a>
-                <a href="#lorem3">Lorem ipsum</a>
-                <a href="#lorem4">Lorem ipsum</a>
-                <a href="#lorem5">Lorem ipsum</a>
-              </ul>
+              <div class="toc">
+                <p class="toc__title">Contents</p>
+                <ol class="toc__list">
```

Comment:
I was following what was done in the Style Guide which is surely a demo of how the code should be written? 
![screen shot 2016-06-16 at 10 16 59](https://cloud.githubusercontent.com/assets/1654845/16111719/ba0809cc-33ab-11e6-9321-58897738a592.png)